code/public/css/app.css

```diff
@@ -11172,6 +11172,10 @@ label.panel-block:hover {
   padding: 3rem 1.5rem 6rem;
 }
 
+html {
+  background: #b2ddfa;
```

Comment:
wydaje mi sie ze to nie powinno byc w githubie. (albo katalog resources, nigdy nie pamietam ktory to jest ten w ktorym piszesz, a ktory to jest ten wynikowy)

Reply:
ok, `public` powinno byc w `.gitignore`